lib/enigma.rb

```diff
@@ -1,8 +1,40 @@
+require 'date'
 class Enigma
+  def rand_key
```

Comment:
This method will generate a random key for our cipher. It's a bit short and will be in every cipher so it could be a good candidate for an instance variable.